bin/helm-deploy

```diff
@@ -41,6 +37,18 @@ if [ "${HELM_TEMPLATE_ONLY}" != "true" ]; then
   helm init --client-only
 fi
 
+detect_tiller_namespace() {
+    # If there is more than one tiller, then auto-detection will not work.
+    mapfile -t TILLERS < <(kubectl get deploy --all-namespaces -l app=helm,name=tiller | awk 'NR > 1 {print $1}')
+    if [ ${#TILLERS[@]} -gt 1 ]; then
+      echo "Multiple TILLERS were found.  This is not supported by auto-detection.  Please specify TILLER_NAMESPACE in your config."
```

Comment:
Only suggestion (non-blocking) is to list the namespaces in which tiller was found - so the user can adjust without as much investigation.

Reply:
👍 Done.